cmake/kokkos_tribits.cmake

```diff
@@ -148,15 +148,20 @@ ELSE()
   CMAKE_PARSE_ARGUMENTS(PARSE
     ""
     ""
-    "SOURCES;CATEGORIES"
+    "SOURCES;CATEGORIES;ARGS"
     ${ARGN})
   VERIFY_EMPTY(KOKKOS_ADD_EXECUTABLE_AND_TEST ${PARSE_UNPARSED_ARGUMENTS})
   KOKKOS_ADD_TEST_EXECUTABLE(${ROOT_NAME}
     SOURCES ${PARSE_SOURCES}
   )
+  IF(DEFINED PARSE_ARGS)
+    STRING(REPLACE ";" " " PARSE_ARGS_STR "${PARSE_ARGS}")
+    SET(OPTIONAL_ARGS CMD_ARGS "${PARSE_ARGS_STR}")
```

Comment:
We should be able to forward these without the ";" replace, yeah?

Reply:
You're right.  Fixed.